data/governments.txt

```diff
@@ -559,8 +559,11 @@ government "Hai (Friendly Unfettered)"
 		"Korath" -.01
 		"Hai" .01
 		"Hai (Wormhole Access)" .01
-		"Hai (Unfettered)" 1
 		"Elenctic Commune" .01
+		"Wanderer" -1e5
+
+		"Hai (Unfettered)" 1e-5
```

Comment:
the idea with this was only that they would be hostile if you attacked the uhai too much, but here scratching them will mean ending the campaign unless there's smth that I'm missing

Reply:
No, if the affinity is less than .05, it won't affect the reputation after you leave the system.

Reply:
just set it at .01 like everyone?

Reply:
Yes.

Reply:
I have fixed this.